Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -84,6 +84,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $em->getConnection()->connect($input->getOption('shard'));
         }
 
+        if ($input->getOption('tag')) {
+            $this->fixturesLoader->setTag($input->getOption('tag'));
```

Comment:
This should not be stored on the loader but rather just be passed to `getFixtures` which does the filtering anyways. No need for adding the tag to the state of the object.